Tasks/GooglePlayReleaseV4/modules/googleutil.ts

```diff
@@ -259,12 +259,45 @@ export async function uploadDeobfuscation(
         tl.debug('returned: ' + JSON.stringify(res));
         return res.data;
     } catch (e) {
-        tl.debug(`Failed to upload deobfuscation file ${mappingFilePath}`);
-        tl.debug(e);
         throw new Error(tl.loc('CannotUploadDeobfuscationFile', mappingFilePath, e));
     }
 }
 
+/**
+ * Uploads a native debugs symbols file (debug.zip) for a given package
+ * Assumes authorized
+ * @param mappingFilePath the path to the file to upload
+ * @param packageName unique android package name (com.android.etc)
+ * @param versionCode version code of uploaded APK or AAB
+ * @returns deobfuscationFiles A promise that will return result from uploading a deobfuscation file
+ *                          { deobfuscationFile: { symbolType: string } }
+ */
+export async function uploadNativeDeobfuscation(
+    edits: pub3.Resource$Edits,
+    mappingFilePath: string,
+    packageName: string,
+    versionCode: number
+): Promise<pub3.Schema$DeobfuscationFilesUploadResponse> {
+    const requestParameters: pub3.Params$Resource$Edits$Deobfuscationfiles$Upload = {
+        deobfuscationFileType: 'nativeCode',
+        packageName: packageName,
+        apkVersionCode: versionCode,
+        media: {
+            body: fs.createReadStream(mappingFilePath),
+            mimeType: ''
+        }
+    };
+
+    try {
+        tl.debug('Request Parameters: ' + JSON.stringify(requestParameters));
+        const res = await edits.deobfuscationfiles.upload(requestParameters, { onUploadProgress });
```

Comment:
Could you please set type for `res` variable?

Reply:
The response type here is long and doesn't add too much to code readability: `googleapis.Common.GaxiosResponse<googleapis.androidpublisher_v3.Schema$DeobfuscationFilesUploadResponse>`.
In all other functions of this module, `res` variable doesn't have an explicit type. Consistency is another reason to keep this type implicit.